components/ArticleMenu/ArticleMenu.tsx

```diff
@@ -188,6 +189,13 @@ const ArticleMenu = ({
                     </a>
                   </Menu.Item>
                 ))}
+                <ReportComments
+                  postTitle={postTitle}
+                  name={postUsername}
+                  postId={postId}
+                  postUsername={postUsername}
+                  postUrl={postUrl}
+                />
```

Comment:
This is the issue. We could possibly put the component as a button below the bookmark?
Something like this.
![Screenshot from 2023-10-17 20-51-17](https://github.com/codu-code/codu/assets/107807466/1d65a2e5-fff9-4685-b1d5-4274fe6fa102)
Maybe that is not the look you want, so im open to suggestions.

Reply:
I was thinking of adding it as an option to the 3 dots menu: 
![Screenshot 2023-10-18 at 07 50 20](https://github.com/codu-code/codu/assets/12615742/edfd8d37-6f68-4f41-a7ee-4939903efd24)